RequestService/RequestService.Repo/Repository.cs

```diff
@@ -2155,5 +2156,189 @@ public async Task<Chart> GetActivitiesByMonth(int groupId)
             
             return result;
         }
+
+        public async Task<Chart> RequestVolumeByDueDateAndRecentStatus(int groupId)
```

Comment:
I'm not convinced a Repository method should be returning a Chart type; I'd prefer to see much of the logic below in a higher layer.